app/controllers/organisations_controller.rb

```diff
@@ -21,7 +21,8 @@ def create
       redirect_to root_path, notice: "#{@organisation.name} created"
     else
       @register_organisations = Organisation.fetch_organisations_from_register
-
+      
+      @hide_sidebar = true
```

Comment:
Could you add a feature test for this?

Reply:
Test added. Have also added a test to ensure the sidebar doesn't appear when the form is first displayed as the same logic was untested there.